e2e-tests/cypress/e2e/journeys/insurance/account/create/your-details/account-create-your-details-validation-account-already-exists.spec.js

```diff
@@ -59,7 +59,7 @@ context('Insurance - Account - Create - Your details page - form validation - As
     const value = mockAccount.email;
     const fieldIndex = 0;
     const TOTAL_REQUIRED_FIELDS = 1;
-    const expectedMessage = YOUR_DETAILS_ERROR_MESSAGES[EMAIL].ACCOUNT_ALREADY_EXISTS;
+    const expectedMessage = String(YOUR_DETAILS_ERROR_MESSAGES[EMAIL].ACCOUNT_ALREADY_EXISTS);
```

Comment:
but this error message is already a string?

Reply:
SonarCloud does not think so because it's a dynamic object structure. e.g here, if the string that is `EMAIL`,  is not found in `YOUR_DETAILS_ERROR_MESSAGES`, it'll return `[Object Object]. This is what it flags so wrapping into a string fixes this concern.

Reply:
I guess it is counter productive in a test since that Error message will always exist for the test.  Would make sense in a non-test file

Reply:
I do agree it feels strange - regardless of if it's in a test or not; but it is a genuine concern, e.g if the `EMAIL` constant is changed to have a value of eg `emailAddress`, this message would not return a string.